omod/src/main/webapp/resources/scripts/patientsearch/patientSearchWidget.js

```diff
@@ -770,6 +774,7 @@ function PatientSearchWidget(configuration){
     // see https://issues.openmrs.org/browse/RA-1404 for potential use cases
     jq('#patient-search-form').on('search:clear', function() {
         clearSearch();
+        jq('#patient-search-form').after(spinnerImage);
```

Comment:
What do you think about instead adding this line here, we make it more explicit the and add a new "start" event, like:

```
jq('patient-search-form').on('search:start', function() {
            jq('#patient-search-form').after(spinnerImage);
}
```

and then in the fingerprintSearchController we do:

```
jq('#patient-search-form').trigger('search:clear')

jq('#patient-search-form').trigger('search:start')
```

This makes it a little explicity that "clearing" the search box and rendering the spinner are two separate events that we just happen to want to do at the same time.

Reply:
@mogoodrich yes, I had this idea, I wanted to propose this to you.